JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/definitions/JavaExpansionMemberVisitor.java

```diff
@@ -218,9 +218,14 @@ public Void visitSetter(SetterMember member) {
 	@Override
 	public Void visitOperator(OperatorMember member) {
 		final JavaCompilingMethod javaMethod = class_.getMethod(member);
+		if (!javaMethod.compile) {
+			return null;
+		}
+
 		final MethodMember methodMember = new MethodMember(member.position, member.definition, member.getEffectiveModifiers(), javaMethod.compiled.name, member.header);
 		methodMember.body = member.body;
 		methodMember.annotations = member.annotations;
+		class_.addMethod(methodMember, javaMethod);
```

Comment:
Since we delegate the operator call to the method call, we need the addMethod, otherwise we NPE since the new MethodMember was never registered.
Should we keep it this way or instead look for a way to extract the common functionality so that we can call the logic with the original OperatorMember?

Reply:
This should be handled during a preparation phase somehow